linode/manager
# chore: Fix Storybook build and add build storybook in CI

## Description 📝

- Fix Storybook build errors
- Add a Github Action that will allow us to manually deploy storybook

## How to test 🧪
- Verify that Storybook builds without errors
  - `yarn workspace linode-manager run build-storybook`
- Optionally test the production build
  - `yarn serve /Users/bnussman/Development/manager/packages/manager/storybook-static`
